src/test/java/com/stripe/functional/FileTest.java

```diff
@@ -4,8 +4,8 @@
 
 import com.stripe.BaseStripeTest;
 import com.stripe.exception.StripeException;
-import com.stripe.model.FileUpload;
-import com.stripe.model.FileUploadCollection;
+//import com.stripe.model.File;
```

Comment:
It looks like you can into problems with a naming conflict on `File` or something? Maybe just removed this commented out import if it's the easiest thing.